modules/auxiliary/dos/http/ua_parser_js_redos.rb

```diff
@@ -0,0 +1,108 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Dos
+
+  def initialize
+    super(
+      'Name'           => 'ua-parser-js npm module - Regular Expression Denial of Service',
+      'Description'    => %q{
+        This module exploits a Regular Expression Denial of Service vulnerability in the npm module "ua-parser-js". Server-side applications that use "ua-parser-js" for parsing the browser user-agent string will be vulnerable if they call the "getOS" or "getResult" functions. This vulnerability was fixed as of version 0.7.16.
+      },
+      'References'     =>
+        [
+          ['CWE', '400'],
```

Comment:
As per comment in PR, was the project notified of the issue?  That would give something to point back to.